### DISCUSSION TITLE
Login help needed: Changed user account to organization in GitHub. Can not login to CodeMagic account anymore

### DISCUSSION THREAD

**Discussion by ryanoberoi** [codemagic-ci-cd/codemagic-docs#1895] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

GitHub Account 1 - use to sign into CodeMagic
Convert account to organization and transfer ownership to GitHub Account 2

Unable to login to Github as Account 1.
Unable to login to CodeMagic.

### Expected results

Was expecting some way to change the to username / password based login in CodeMagic. or to change the email account used / and re-login with a different GitHub Account.

Note: The GitHub integration works correctly, just the login piece in missing.

### Actual results

Have a dangling GitHub account 1 which can't be used to login to CodeMagic.

**Comment by kulakarla** [comment]
Hello @ryanoberoi 
What do you mean by this: "Note: **The GitHub integration works correctly**, just the login piece in missing."?

**Comment by kulakarla** [comment]
I'm tenatively closing this thread due to inactivity.
